contracts/QuestFactory.sol

```diff
@@ -54,6 +56,17 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         protocolFeeRecipient = protocolFeeRecipient_;
     }
 
+    /// @dev Create either an erc20 or erc1155 quest
```

Comment:
we should also add that it's whitelisted deployment only

Reply:
done.